<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>805</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>611</width>
      <height>131</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="digitCount">
     <number>12</number>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>611</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="3">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer_9">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="plusButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>55</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="multiplyButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>55</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="minusButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>55</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="divideButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>55</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="text">
          <string>รท</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_10">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="sevenButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>55</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fourButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>55</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="oneButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>55</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="zeroButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>55</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="eightButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>55</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="fiveButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>55</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="twoButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>55</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="decimalButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>55</pointsize>
          </font>
         </property>
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="4">
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>402</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>55</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#clearButton{

background: rgb(252, 233, 79)
}</string>
         </property>
         <property name="text">
          <string>AC</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="nineButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>55</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sixButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>55</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="threeButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>55</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="equalButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>55</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>CrossCursor</cursorShape>
         </property>
         <property name="text">
          <string>=</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>669</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
